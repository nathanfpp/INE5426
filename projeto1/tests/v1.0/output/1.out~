int array: a (size: 10)
int var: i = 0
int ref var: mypointer = 0, aa
int ref  ref  ref  ref  array: b (size: 10)

