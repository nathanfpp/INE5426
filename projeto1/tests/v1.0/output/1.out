int array: a (size: 10)
int var: i = 0
int ref var: mypointer = 0, aa

